nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -422,6 +469,14 @@ in
         '';
       };
 
+      recommendedBrotliSettings = mkOption {
+        default = false;
+        type = types.bool;
+        description = lib.mdDoc ''
+          Enable recommended brotli settings.
```

Comment:
Can you add here it is required to *add* the brotli module to `additionalModules` ? Otherwise people using nixos-search will get confused.

Reply:
Updated description.